pkgs/by-name/lx/lxc/package.nix

```diff
@@ -51,16 +51,37 @@ stdenv.mkDerivation (finalAttrs: {
 
     # fix linking
     ./4428.diff
+
+    # Fix hardcoded path of lxc-user-nic
+    # This is needed to use unprivileged containers
+    ./user-nic.diff
   ];
 
   mesonFlags = [
-    "-Dinstall-init-files=false"
+    "-Dinstall-init-files=true"
     "-Dinstall-state-dirs=false"
     "-Dspecfile=false"
+    "-Dusernet-config-path=/etc/lxc/lxc-usernet"
+    "-Ddistrosysconfdir=${placeholder "out"}/etc/default"
+    "-Dsystemd-unitdir=${placeholder "out"}/lib/systemd/system"
     # re-enable when fixed https://github.com/lxc/lxc/issues/4427
     # "-Dtools-multicall=true"
   ];
 
+  # /run/current-system/sw/share
+  postInstall = ''
+    substituteInPlace $out/etc/default/lxc --replace "$out/etc/default" "/etc/default"
```

Comment:
```suggestion
    substituteInPlace $out/etc/default/lxc --replace-fail "$out/etc/default" "/etc/default"
```

Please use `replace-fail` on all of these.

Reply:
I'd also like to not create `default` and store these in the `lxc` directory if possible.